src/Widgets/AppContextMenu.vala

```diff
@@ -82,24 +80,36 @@ public class Slingshot.AppContextMenu : Gtk.Menu {
             });
         }
 
-        var dock = Backend.Dock.get_default ();
-        if (dock.dbus != null) {
+        if (Environment.find_program_in_path ("io.elementary.dock") != null) {
             if (get_children ().length () > 0) {
                 add (new Gtk.SeparatorMenuItem ());
             }
-
```

Comment:
Is there any reason why you removed this?

We prefer having a newline after closing brackets so I'd like you to keep this as before if there are no special reasons.